packages/plugins/custom-fields/admin/src/components/ColorPicker/ColorPickerInput/index.js

```diff
@@ -0,0 +1,198 @@
+  }
+
+  .react-colorful__pointer {
+    width: ${({ theme }) => theme.spaces[3]};
+    height: ${({ theme }) => theme.spaces[3]};
+  }
+
+  .react-colorful__saturation {
+    border-radius: ${({ theme }) => theme.spaces[1]};
+    border-bottom: none;
+  }
+
+  .react-colorful__hue {
+    border-radius: 10px;
+    height: ${({ theme }) => theme.spaces[3]};
+    margin-top: ${({ theme }) => theme.spaces[2]};
+  }
+`;
+
+const ColorPickerToggle = styled(BaseButton)`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  svg {
+    width: ${({ theme }) => theme.spaces[2]};
+    height: ${({ theme }) => theme.spaces[2]};
+  }
+
+  svg > path {
+    fill: ${({ theme }) => theme.colors.neutral500};
+    justify-self: flex-end;
+  }
+`;
+
+const ColorPickerPopover = styled(Popover)`
+  padding: ${({ theme }) => theme.spaces[1]};
+  min-height: 260px;
+`;
+
+const ColorPickerInput = ({
+  attribute,
+  description,
+  disabled,
+  error,
+  intlLabel,
+  labelAction,
+  name,
+  onChange,
+  required,
+  value,
+}) => {
+  const [showColorPicker, setShowColorPicker] = useState(false);
+  const colorPickerButtonRef = useRef();
+  const { formatMessage } = useIntl();
+  const color = value || '#ffffff';
+  const styleUppercase = { textTransform: 'uppercase' };
+
+  const handleBlur = (e) => {
+    e.preventDefault();
+
+    if (!e.currentTarget.contains(e.relatedTarget)) {
+      setShowColorPicker(false);
+    }
+  };
+
+  return (
+    <Field
+      name={name}
+      id={name}
+      // GenericInput calls formatMessage and returns a string for the error
+      error={error}
+      hint={description && formatMessage(description)}
+    >
+      <Stack spacing={1}>
+        <Flex>
+          <FieldLabel action={labelAction} required={required}>
+            {formatMessage(intlLabel)}
+          </FieldLabel>
+        </Flex>
+        <ColorPickerToggle
+          ref={colorPickerButtonRef}
+          aria-label="Color picker toggle"
+          aria-controls="color-picker-value"
+          aria-haspopup="dialog"
+          aria-expanded={showColorPicker}
+          aria-disabled={disabled}
+          disabled={disabled}
+          onClick={() => setShowColorPicker(!showColorPicker)}
+        >
+          <Flex>
+            <ColorPreview color={color} />
+            <Typography
+              style={styleUppercase}
+              textColor={value ? null : 'neutral600'}
+              variant="omega"
+            >
+              {color}
+            </Typography>
+          </Flex>
+          <CarretDown aria-hidden />
+        </ColorPickerToggle>
+        {showColorPicker && (
+          <ColorPickerPopover
+            onBlur={handleBlur}
+            role="dialog"
+            source={colorPickerButtonRef}
+            spacing={4}
+          >
+            <FocusTrap onEscape={() => setShowColorPicker(false)}>
+              <ColorPicker
+                color={color}
+                onChange={(hexValue) =>
```

Comment:
Did `onChange={onChange}` not work?

Reply:
It did not, if I remember correctly but I'll try again